### ISSUE TITLE
DemoParser.Exception: EntityNotFound

### ISSUE THREAD

**Issue by TomQWERTY** [pnxenopoulos/awpy#303]
I was trying to test awpy 2.0 dev version with some of my premier and matchmaking demos. While trying to parse some of them, I was getting error message saying:
"Traceback (most recent call last):
  File "f:\...\awpy2test.py", line 3, in <module>
    parsed = parse_demo(file=r"file_name.dem")          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\username\AppData\Local\Programs\Python\Python311\Lib\site-packages\awpy\parser\demoparser.py", line 374, in parse_demo
    parsed_round_events = parser.parse_events(
                          ^^^^^^^^^^^^^^^^^^^^
DemoParser.Exception: EntityNotFound"

My code:
from awpy import parse_demo
parsed = parse_demo(file=r"file_name.dem")
print(parsed["header"])

I got few different demos that are throwing this error, some newer, some older, so here is one of them: https://drive.google.com/file/d/1RqA2JcV_GoPBc7kk3AzifacPUq0eWb99/view?usp=drive_link

This demo was recorded today (10/31/2023). Also maybe worth mentioning, that I have another demo that was recorded today too, and it doesn't give such an error, it is parsed fine.

**Comment by pnxenopoulos** [comment]
Is this a POV demo?

**Comment by TomQWERTY** [comment]
> Is this a POV demo?

If POV demo is a demo recorded with typing "record *fileName*" in console, then yes

**Comment by JanEricNitschke** [comment]
Yes, those are POV demos.

**Comment by pnxenopoulos** [comment]
Closing, same response as in https://github.com/pnxenopoulos/awpy/issues/302

TLDR: Old POV demos have historically been tricky :) so we probably won't support these, but we want to support POV demos going forward, particularly for recent POV demos.
